### ISSUE TITLE
Best parameter to reproduce the best results

### ISSUE THREAD

**Issue by Jorigorn** [happywwy/Recursive-Neural-Conditional-Random-Field#3]
Hi, thanks for sharing the code, nice work.
I have set up the environment and get it running.
But the test results seems not so good.

What's the meaning of "error, 2.41666005858", I mean, how to calculate this error?

I am trying to apply LSTM/Memory netowrk model instead of CRF alone, on top of DP-RNN output, or some Combination/ensemble model , appreciate for any advices.

Thanks a lot. :)

here is the results in finallog:

epoch: 0 inst_ind: 1 error, 8.25400929856 time = 0.267558097839 sec
epoch: 0 inst_ind: 2 error, 1.16211643958 time = 0.280063152313 sec
epoch: 0 inst_ind: 3 error, 11.0444722865 time = 0.215932846069 sec
epoch: 0 inst_ind: 4 error, 1.13270115969 time = 0.207509040833 sec
epoch: 0 inst_ind: 5 error, 3.00617358466 time = 0.19717502594 sec
epoch: 0 inst_ind: 6 error, 5.89935760121 time = 0.23672914505 sec
epoch: 0 inst_ind: 7 error, 4.59122429141 time = 0.208659887314 sec
epoch: 0 inst_ind: 8 error, 23.3169124487 time = 0.225489139557 sec
epoch: 0 inst_ind: 9 error, 4.84739099586 time = 0.19690489769 sec
epoch: 0 inst_ind: 10 error, 3.34838658274 time = 0.214997053146 sec
epoch: 0 inst_ind: 11 error, 2.41666005858 time = 0.193145036697 sec

update1:
I found the parameter setting in the paper, I tried to increase the epoch number, even to 200, the accuracy seems don't have much difference.

Thanks, John.

**Comment by happywwy** [comment]
The word embeddings pretrained are generated from yelp dataset for restaurant and amazon reviews for laptop. Since the embedding occupy too much space, I did not upload it, but it should be producible. The errors are calculated based on the loss function.
